src/hb-ot-layout.h

```diff
@@ -37,6 +37,11 @@
 
 HB_BEGIN_DECLS
 
+/*
+ * The maximum number of times a lookup can be applied during shaping.
+ * Used to limit the number of iterations of the closure algorithm.
+ */
+#define HB_CLOSURE_MAX_STAGES 8
```

Comment:
Please move this to `hb-ot-layout-common-private.hh` after the couple of other defines there, and also guard with `ifndef` like them.

Reply:
Done.